package.json

```diff
@@ -2,15 +2,16 @@
   "name": "@seneca/debug",
   "version": "1.0.1",
   "description": "Seneca plugin for debugging Seneca-based systems.",
-  "main": "debug.js",
+  "main": "index.js",
   "scripts": {
     "watch": "webpack --watch",
     "test": "lab -P test -t 60 -I URL,URLSearchParams",
-    "prettier": "prettier --write --no-semi --single-quote *.js lib/*.js test/*.js",
     "coveralls": "lab -s -P test -r lcov | coveralls",
     "clean": "rm -rf node_modules yarn.lock package-lock.json",
     "repo-tag": "REPO_VERSION=`node -e \"console.log(require('./package').version)\"`; echo TAG: v$REPO_VERSION; git commit -a -m v$REPO_VERSION; git push; git tag v$REPO_VERSION; git push --tags;",
-    "repo-publish": "npm run repo-tag; npm publish --access public"
+    "repo-publish": "npm run repo-tag; npm publish --access public",
```

Comment:
see https://github.com/senecajs/seneca-flow/blob/main/package.json#L18 for the more recent set of tasks that are used for publication